setup.py

```diff
@@ -57,7 +57,7 @@ def read_version():
                       'pandas', 'Pillow', 'h5py'],
     extras_require={
         'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock',
-                 'sagemaker==1.19.1', 'tensorflow<2.0', 'docker-compose', 'botocore>=1.12.140'],
+                 'sagemaker==1.19.1', 'tensorflow', 'docker-compose', 'botocore>=1.12.140'],
```

Comment:
sagemaker and botocore are significantly behind the latest version. Is there any package that is dependent on these particular versions?